rest_api/rest_api/utils.py

```diff
@@ -64,13 +70,15 @@ def get_openapi_specs() -> dict:
     """
     Used to autogenerate OpenAPI specs file to use in the documentation.
 
-    See `docs/_src/api/openapi/generate_openapi_specs.py`
+    See `.github/utils/generate_openapi_specs.py`
```

Comment:
We should also add a short mention of why we're passing `servers` to `get_openapi` for future reference.